### DISCUSSION TITLE
View indexes

### DISCUSSION THREAD

**Discussion by williambuchanan2** [sqlitebrowser/sqlitebrowser#3720] [General]
Hi all

Can someone tell me how I can view indexes? I see in the wiki that you can 'delete indexes', which implies there must be some way of viewing them, but I can't work out how...

Thanks

**Comment by justinclift** [comment]
In theory, indices are shown in the "Database Structure" tab like the **candidate_names_idx** index in the screenshot below:

![image](https://github.com/user-attachments/assets/b77be88f-51b4-488b-b0ca-bdde9a20cc6e)

**Reply by williambuchanan2** [reply]
Ah I see them now. I expected them to be with the table. Didn't even think of looking for them in a different place. Seems a bit strange...

**Reply by justinclift** [reply]
No worries.  Hopefully it's not too hard to get used to. :smile:
